core/src/View/MDSpan/Kokkos_MDSpan_Layout.hpp

```diff
@@ -24,6 +24,7 @@ static_assert(false,
 
 #include "Kokkos_MDSpan_Extents.hpp"
 #include <View/Kokkos_ViewDataAnalysis.hpp>
+#include <Kokkos_Abort.hpp>
```

Comment:
Now that there's no abort, is this include still necessary?

Reply:
OOps, fixed!